app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -230,28 +239,46 @@ private fun ColumnScope.ImportTextFields(transferMessage: GetSetCallbacks<String
 }
 
 @Composable
-private fun ColumnScope.EmailAddressesTextFields(selectedTransferType: () -> TransferTypeUi) {
+private fun ColumnScope.EmailAddressesTextFields(modifier: Modifier, selectedTransferType: () -> TransferTypeUi) {
```

Comment:
I don't think you need the `ColumnScope`

Reply:
same